api/tests/routes/pro/get_offerer_addresses_test.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.users import factories as users_factories
+
+
+@pytest.mark.usefixtures("db_session")
+def test_get_offerer_addresses_success(client):
+    user_offerer = offerers_factories.UserOffererFactory()
+    pro = user_offerer.user
+    offerer = user_offerer.offerer
+
+    offerer_address_1 = offerers_factories.OffererAddressFactory(
+        offerer=offerer,
+        label="1ere adresse",
+        address__street="1 boulevard Poissonnière",
+        address__postalCode="75000",
```

Comment:
C'est vraiment pour pinailler :)
```suggestion
        address__postalCode="75002",
```